src/components/Filecard.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import { Icon } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import grid from './css/grid-view.css'
```

Comment:
always keep one line space between foreign libraries import and custom js and css file import